web3swift/src/Account/EthereumKeyStorage.swift

```diff
@@ -25,7 +23,7 @@ public enum EthereumKeyStorageError: Error {
     case failedToDelete
 }
 
-public class EthereumKeyLocalStorage: EthereumSingleKeyStorageProtocol {
+public class EthereumKeyLocalStorage: EthereumKeyStorageProtocol {
```

Comment:
We still need `SingleKeyStorageProtocol` as it covers the usecase where the user doesn't care about multiple addresses, and doesn't know the address until private key is loaded.